pmd-core/src/main/java/net/sourceforge/pmd/PMDConfiguration.java

```diff
@@ -210,6 +211,23 @@ public LanguageVersionDiscoverer getLanguageVersionDiscoverer() {
         return languageVersionDiscoverer;
     }
 
+    /**
+     * Get the LanguageVersion specified as input
```

Comment:
Please mention that this may be null and what happens when it is

Reply:
Agreed, added more to the comment